website/content/en/preview/concepts/nodeclasses.md

```diff
@@ -1473,6 +1502,8 @@ status:
       - arm64
 ```
 
+Note that Karpenter can only discover architecture based requirements unless using an alias, in which case it can discover additional requirements (e.g. GPU / Accelerator support).
```

Comment:
the language confused me a bit, one suggestion
```suggestion
Note that Karpenter can only discover architecture based requirements. The one exception is using alias, in which case it can discover additional requirements (e.g. GPU / Accelerator support).
```